jupyter_server/services/sessions/sessionmanager.py

```diff
@@ -268,15 +276,34 @@ async def create_session(
         self._pending_sessions.remove(record)
         return result
 
-    def get_kernel_env(self, path):
-        """Return the environment variables that need to be set in the kernel"""
+    async def get_kernel_env(self, path: str, name: Optional[str] = None) -> Dict[str, str]:
+        """Return the environment variables that need to be set in the kernel
+
```

Comment:
Similar question.  Seems like we should document all parameters or leave things alone.